Add profile prod with postgres and change default to H2

normanjurisch opened the pull request:
added application-prod.yaml for Postgres support
changed default profile to H2

normanjurisch committed:
H2 support in application.yaml
.gitignore-change for H2 DB files
added application-prod.yaml for Postgres support

rhea-0b1 commented:
Nice and thanks for your contribution my friend  :) 

Can you please rename the spring profiles and create the following two:
- application-h2.yaml
- application-postgres.yaml

normanjurisch commented:
What should be default in application.yaml?

rhea-0b1 commented:
Just the Camunda and spring security configuration

normanjurisch committed:
Separate yaml configuration files for h2 and postgres.

Choose h2 or postgres profile in your run configuration in favour of your environment
